pkg/encoder/token_encoder.go

```diff
@@ -0,0 +1,40 @@
+package encoder
+
+import (
+	"github.com/openfga/openfga/pkg/encrypter"
+)
+
+type TokenEncoder struct {
+	encrypter encrypter.Encrypter
+	encoder   Encoder
+}
+
+var _ Encoder = (*TokenEncoder)(nil)
+
+// NewTokenEncoder constructs an Encoder that includes an encrypter and encoder.
```

Comment:
Did you mean?

// NewTokenEncoder constructs a TokenEncoder with the provided encrypter and encoder.

Reply:
Yes, that's better. Thanks!